rinex/src/header.rs

```diff
@@ -1435,25 +1434,25 @@ impl Header {
             writeln!(f, "{}", fmt_rinex(&descriptor, "# / TYPES OF DATA"))?;
 
             // possible timescale
-            if let Some(ts) = clocks.timescale {
+            if let Some(ts) = clock.timescale {
                 writeln!(
                     f,
                     "{}",
                     fmt_rinex(&format!("   {:x}", ts), "TIME SYSTEM ID")
                 )?;
             }
 
-            // possible agency
-            if let Some(agency) = &clocks.agency {
-                writeln!(
-                    f,
-                    "{}",
-                    fmt_rinex(
-                        &format!("{:<5} {}", agency.code, agency.name),
-                        "ANALYSIS CENTER"
-                    )
-                )?;
-            }
+            //// possible agency
```

Comment:
Intentional?

Reply:
not really, formatting this type of file is work in progress, but I'd rather contribute it in following PR and have these current improvements merged